src/spikeinterface/sorters/launcher.py

```diff
@@ -68,10 +67,18 @@ def run_sorter_jobs(job_list, engine="loop", engine_kwargs={}, return_output=Fal
         The engine to run the list.
         * "loop" : a simple loop. This engine is
     engine_kwargs : dict
-
-    return_output : bool, dfault False
+        In the case of engine="slum", possible kwargs are:
+        - tmp_script_folder: str, default None
+            the folder in which the job scripts are created. Default: directory created by
+            the `tempfile` library
+        - sbatch_executable_path: str, default 'sbatch'
```

Comment:
This is cool, out of interest what is the use case? will `sbatch` alone not always work?

Reply:
Just in case sbatch is not in the PATH. Unlikely but why not ?

Reply:
I can see why this would be useful but I think in terms of codebase maintainability it would be better that the user handles this on their end by adding `sbatch` to PATH, given that it is (I think?) a real edge case. Otherwise, if this is a general pattern that should be adopted it will require adding such arguments everywhere (e.g. if a sorter uses `matlab`, it will have to expose a path to the `matlab` executable, similar for anything that calls a system executable).

Reply:
That's fair